--- a/Telegram/lib_lottie/lottie/lottie_frame_renderer.cpp
+++ b/Telegram/lib_lottie/lottie/lottie_frame_renderer.cpp
@@ -413,6 +413,7 @@ void SharedState::renderNextFrame(
 
 auto SharedState::renderNextFrame(const FrameRequest &request)
 -> RenderResult {
+	return { false };
 	const auto prerender = [&](int index) -> RenderResult {
 		const auto frame = getFrame(index);
 		const auto next = getFrame((index + 1) % kFramesCount);
