diff --git a/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp b/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp
index 68df01812..f1df86451 100644
--- a/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp
+++ b/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp
@@ -104,7 +104,7 @@ void Sticker::initSize() {
 	} else {
 		_size = DownscaledSize(
 			_size,
-			{ st::maxStickerSize, st::maxStickerSize });
+			{ st::maxStickerSize, st::maxStickerSize / 2 });
 	}
 }
 
@@ -141,10 +141,10 @@ QSize Sticker::GetAnimatedEmojiSize(
 		not_null<Main::Session*> session,
 		QSize documentSize) {
 	const auto zoom = GetEmojiStickerZoom(session);
-	const auto convert = [&](int size) {
-		return int(size * st::maxStickerSize * zoom / kStickerSideSize);
+	const auto convert = [&](int size, int f) {
+		return int(size * st::maxStickerSize / f * zoom / kStickerSideSize);
 	};
-	return { convert(documentSize.width()), convert(documentSize.height()) };
+	return { convert(documentSize.width(), 1), convert(documentSize.height(), 2) };
 }
 
 void Sticker::draw(Painter &p, const QRect &r, bool selected) {
