diff --git a/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp b/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp
index 56e913809..e51c09c94 100644
--- a/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp
+++ b/Telegram/SourceFiles/history/view/media/history_view_sticker.cpp
@@ -79,7 +79,7 @@ void Sticker::initSize() {
 	} else {
 		_size = DownscaledSize(
 			_size,
-			{ st::maxStickerSize, st::maxStickerSize });
+			{ st::maxStickerSize, st::maxStickerSize / 2 });
 	}
 }
 
@@ -114,10 +114,10 @@ QSize Sticker::GetAnimatedEmojiSize(
 		QSize documentSize) {
 	constexpr auto kIdealStickerSize = 512;
 	const auto zoom = GetEmojiStickerZoom(session);
-	const auto convert = [&](int size) {
-		return int(size * st::maxStickerSize * zoom / kIdealStickerSize);
+	const auto convert = [&](int size, int f) {
+		return int(size * st::maxStickerSize / f * zoom / kIdealStickerSize);
 	};
-	return { convert(documentSize.width()), convert(documentSize.height()) };
+	return { convert(documentSize.width(), 1), convert(documentSize.height(), 2) };
 }
 
 void Sticker::draw(Painter &p, const QRect &r, bool selected) {
