diff --git a/Telegram/SourceFiles/ui/chat/chat.style b/Telegram/SourceFiles/ui/chat/chat.style
index 4364fca61..183f967bb 100644
--- a/Telegram/SourceFiles/ui/chat/chat.style
+++ b/Telegram/SourceFiles/ui/chat/chat.style
@@ -29,13 +29,13 @@ defaultMessageBar: MessageBar {
 
 minPhotoSize: 100px;
 minVideoSize: 160px;
-maxMediaSize: 430px;
+maxMediaSize: 220px;
 maxStickerSize: 228px;
-maxGifSize: 320px;
-maxVideoMessageSize: 240px;
+maxGifSize: 160px;
+maxVideoMessageSize: 160px;
 maxSignatureSize: 144px;
 maxWallPaperWidth: 160px;
-maxWallPaperHeight: 240px;
+maxWallPaperHeight: 160px;
 historyThemeSize: size(272px, 176px);
 
 historyMinimalWidth: 380px;
